DFC.ServiceTaxonomy.ContentApproval/Views/ContentsMeta_SummaryAdmin.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@{
+    ContentItem contentItem = Model.ContentItem;
+}
+<div class="tags mr-1">
+    @if (contentItem.Has<ContentApprovalPart>())
+    {
+        ContentApprovalPart contentApprovalPart = contentItem.As<ContentApprovalPart>();
+
+        if (contentApprovalPart.ReviewStatus == ContentReviewStatus.ReadyForReview)
+        {
+            <span class="badge ta-badge font-weight-normal align-text-top" data-toggle="tooltip" title="@(T["Waiting for review"])">
```

Comment:
I know it's not part of the spec, but it might be nice to include the type of review in the title, e.g. 'Waiting for UX review' (and the same for 'In review').